source/declarations.tex

```diff
@@ -2563,13 +2565,23 @@
 
 \pnum
 In a declaration
-\opt{\grammarterm{attribute-specifier-seq}}
 \tcode{T}
 \tcode{D}
 where
 \tcode{D}
-is an unadorned identifier the type of this identifier is
-``\tcode{T}''.
+has the form
+\begin{ncsimplebnf}
+declarator-id \opt{attribute-specifier-seq}
+\end{ncsimplebnf}
+the type of the
+\grammarterm{declarator-id}
+in
+\tcode{D}
+is ``\tcode{T}''.
+The optional
+\grammarterm{attribute-specifier-seq}
```

Comment:
Since this seems to be new text, please follow the "semantic linebreaks" conventions. In particular, don't start a new line when a \xxx LaTeX thing occurs. (These are historic accidents from when we did the standard in troff.)

Reply:
Should I change this for all the modified lines? Or just for new text.

Reply:
As a first step, the usual green blob

then the type of the
\grammarterm{declarator-id}
in

should be on one line, and also the corresponding blob above (maybe that needs two lines).
Also
\tcode{T}
\tcode{D}
should be on one line for the new next.

Please don't wholesale-change the unchanged stuff; that should be part of a different change.